src/common/helpers.js

```diff
@@ -359,3 +359,23 @@ export function numberFormatter(num, digits = 0) {
   }
   return (num / si[i].value).toFixed(digits).replace(rx, '$1') + si[i].symbol;
 }
+
+/**
+ * Subscribe an email to CCDL Emails
+ *
+ * @param {*} EmailAddress
+ */
+
+export const subscribeToMailingList = async email => {
```

Comment:
Do we use this anywhere?

Reply:
good call that should be at the end of the submit for datasetdowloadoptionsform.js